src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -286,14 +286,6 @@ protected override async void FileList_PreviewKeyDown(object sender, KeyRoutedEv
 				FilePropertiesHelpers.ShowProperties(ParentShellPageInstance);
 				e.Handled = true;
 			}
-			else if (e.Key == VirtualKey.Space)
-			{
-				if (!IsRenamingItem && !ParentShellPageInstance.ToolbarViewModel.IsEditModeEnabled)
-				{
-					e.Handled = true;
```

Comment:
This e.Handled = true is very important, it prevents the selection from being reset if multiple items are selected.

Reply:
I don't know what your problem is. Could you explain.

Reply:
It's alright, I worked around this be keeping the code behind.